week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,9 +1,13 @@
 // Declare your function first - call it "add"
-
+function add() {
+  return 124 + 13;
+}
 // Call the function and assign to a variable `sum`
 
+var sum = add();
```

Comment:
Here, you want to pass values into the add function call. So it should be `var sum = add(124, 13);` instead of `var sum = add();`